src/guide/essentials/application.md

```diff
@@ -53,7 +53,7 @@ An application instance won't render anything until its `.mount()` method is cal
 app.mount('#app')
 ```
 
-The content of the app's root component will be rendered inside the container element. The container element itself is not considered part of the app.
+The content of the app's root component will be rendered inside the container element. The container element itself is not considered part of the app. If you're running Vue with a build step, the container element is the div in the `index.html` file at the root of your project. You can change the id of that div from 'app' to any other name. You would have to change the argumnet passed to the `createApp.mount()` method to match the id of the div.
```

Comment:
Good work. Please check, there is a typo error in the word 'Argument'.

Reply:
You're right. I'll fix right away.